docs/source/conf.py

```diff
@@ -166,12 +166,12 @@
     'logo_only': False,
 }
 
-html_logo = '_static/images/lightning_logo-name.svg'
+html_logo = '_images/logos/lightning_logo-name.svg'
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+html_static_path = ['_static', '_images', '_templates']
```

Comment:
```suggestion
html_static_path = ['_images', '_templates']
```
(fixed docs build error)